.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## Asscociated Issue
+ 
+Replace this text with the Jira issue resolved (e.g. CASSANDRA-####) text specifying why this is a trivial patch.
+
+## Author(s) and Reviewers
+
+Patch by <author>; reviewed by <reviewer[s]>
+
+Co-authored-by: author1 <author1@domain.com>, author2 <author2@domain.com>, ..., authorN <authorN@domain.com>
+
+## What the patch does
+
+Replace this text with a detailed commit paragraph of why and how the change was made and relevant context in a paragraph here
+
+##  Recommendations:
+
+- Commits should be squashed to remove intermediate development commit messages.
+- Key commit messages start with the issue number (CASSANDRA-xxxx)
+- if not a trivial change:
```

Comment:
personally I would prefer we don't have this line and leave figuring out if trivial to the committers, so keep line 20 to 22, and remove the indent.  What is trivial for one may not be for another, so defaulting to best practice with committers being able to allow special cases for trivial work.

Reply:
+1 to this.  What is and what is not trivial is up the committer ultimately.